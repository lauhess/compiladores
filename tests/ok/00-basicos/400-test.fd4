let x : Nat = ((
    let a: Nat = 1 in let b: Nat = 2 in a + b) 
    + 2) 

let y : Nat = print "value of x is: " x