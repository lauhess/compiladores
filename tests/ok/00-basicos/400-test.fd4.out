value of x is: 5
